.github/workflows/build-dist.yml

```
@@ -55,13 +59,21 @@
           mv main-windows-latest/main dist/main_windows.exe
           chmod a+x dist/main_*
       - name: Commit and push if changes exist
+        id: commit_and_push
         run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)